packages/create-astro/test/fixtures/not-empty/package.json

```diff
@@ -3,7 +3,7 @@
   "private": true,
   "scripts": {
     "dev": "astro dev",
-    "build": "astro check && astro build",
```

Comment:
Why did you remove astro check? (and the extended tsconfig?)

Reply:
Ah, these files were actually reflecting an incorrect state _after_ some test runs.

We were previously not calling `after(() => resetFixtures())`, but we should have been. These files are now aligned with the "reset" state.